Makefile

```diff
@@ -65,12 +65,11 @@ coverage: ## check code coverage quickly with the default Python
 		$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/bigchaindb.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ bigchaindb
```

Comment:
I don't know what they are tbh. Our hook into the docs build process is via `extensions` in `docs/server/source/conf.py`. I don't recognise any of the above. I think @sbellem is the person to ask.